File: src/test/java/net/questcraft/apitests/MixinTestCase.java
package net.questcraft.apitests;

import net.yakclient.mixin.api.*;
import org.objectweb.asm.tree.InsnList;


@Mixer("net.questcraft.apitests.MixinSourceClassTest")
public class MixinTestCase {
//    private int shadowInt;
//    private String somethingElse;

//    @Shadow
//    public abstract void shadowMethod();
//
//    public abstract void second();
//
//    public static void staticTest() {
//        System.out.println("The current class is this");
//    }

    @Injection(type = InjectionType.BEFORE_RETURN)
    public void printTheString(int test) {
        if (test > 10) System.out.println("THis");
    }
}

File: base/src/test/kotlin/net/yakclient/example/MixinTestCase.kt
package net.yakclient.example

import net.yakclient.mixin.api.Injection
import net.yakclient.mixin.api.Mixer

@Mixer("net.yakclient.example.MixinSourceClassTest")
class MixinTestCase {
    //    private int shadowInt;
    //    private String somethingElse;
    //    @Shadow
    //    public abstract void shadowMethod();
    //
    //    public abstract void second();
    //
    //    public static void staticTest() {
    //        System.out.println("The current class is this");
    //    }
    @Injection
    fun printTheString() {
        val otherString = "clashing?"
        println(otherString)
    }
}